addons/google_calendar/models/calendar.py

```diff
@@ -57,7 +57,7 @@ def _odoo_values(self, google_event, default_reminders=()):
         if google_event.is_cancelled():
             return {'active': False}
 
-        alarm_commands = self._odoo_reminders_commands(google_event.reminders.get('overrides') or default_reminders)
+        alarm_commands = self._odoo_reminders_commands(google_event.reminders.get('overrides', google_event.reminders.get('useDefault') and default_reminders or ()))
```

Comment:
Thanks for your suggestion but I have a few questions :-).

https://developers.google.com/calendar/api/concepts/reminders
"useDefault" is a boolean. 
google_event.reminders.get('overrides', google_event.reminders.get('useDefault')
will either get the overrides list or a boolean 🤔
# Overrides can be set if and only if useDefault is false.
so if you use default, overrides will be empty. That's what we want
in the signature default_reminders=()
so that means that when you sync, it's not actually (). I think this should be the direction to investigate.